### DISCUSSION TITLE
字段标识能动态生成吗？

### DISCUSSION THREAD

**Discussion by manong333** [alibaba/designable#90] [Q&A]
现在的字段标识是自动生成的，有办法能根据后台字段动态匹配生成吗？
![image](https://user-images.githubusercontent.com/76724683/136882925-b401ac32-f914-471c-8fd4-de8674398db6.png)

**Comment by zxbit2011** [comment]
![image](https://user-images.githubusercontent.com/25721936/148522967-b18f387f-8c2f-4a16-8c2c-5ec15244d96a.png)
![image](https://user-images.githubusercontent.com/25721936/148523027-67e7b20b-da6a-498c-a41a-fe6a147c6f92.png)
